README.md

```diff
@@ -1,3 +1,5 @@
+[![Daily Workflow](https://github.com/osrf/buildfarm-tools/actions/workflows/dailyWorkflow.yml/badge.svg)](https://github.com/osrf/buildfarm-tools/actions/workflows/dailyWorkflow.yml)
```

Comment:
Do we actively check this? I had internally the private readme.md as "support tools dashboards" in a tab, would it make sense instead to move this badge there?